tools/docker/bin/monorepo-entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# Check and set PNPM store location
+EXPECTED_STORE="/workspace/tools/docker/data/pnpm-store"
+CURRENT_STORE=$(pnpm config get store-dir)
+
+if [ "$CURRENT_STORE" != "$EXPECTED_STORE" ]; then
+    echo "Setting PNPM store directory to $EXPECTED_STORE"
+    mkdir -p "$EXPECTED_STORE"
+    pnpm config set store-dir "$EXPECTED_STORE"
+fi
+
+# Check if jetpack command is available
+if ! pnpm jetpack --help >/dev/null 2>&1; then
```

Comment:
Slightly simpler since we're using `/bin/bash` is to do
```suggestion
if ! pnpm jetpack --help &>/dev/null; then
```